internal/store/pod.go

```diff
@@ -89,9 +89,99 @@ func podMetricFamilies(allowLabelsList []string) []generator.FamilyGenerator {
 		createPodStatusScheduledFamilyGenerator(),
 		createPodStatusScheduledTimeFamilyGenerator(),
 		createPodStatusUnschedulableFamilyGenerator(),
+		createPodReadinessProbeFamilyGenerator(),
+		createPodLivenessProbeFamilyGenerator(),
+		createPodPrivilegedFamilyGenerator(),
 	}
 }
 
+func createPodPrivilegedFamilyGenerator() generator.FamilyGenerator  {
+	return *generator.NewFamilyGenerator(
+		"kube_pod_container_privileged",
+		"Tells whether the container of the pod is in privilege mode.",
```

Comment:
I order to be consistent with other description messages, I would replace tells with describes:
```
Describes whether the containers readiness check succeeded.
```

Same for the other added metrics.

Reply:
done.